github/ci/prow/files/jobs/nmstate/kubernetes-nmstate-presubmits.yaml

```diff
@@ -93,3 +93,34 @@ presubmits:
               - "/bin/sh"
               - "-c"
               - "automation/check-patch.e2e-okd.sh"
+    - name: pull-kubernetes-nmstate-e2e-ocp
+      skip_branches:
+        - release-\d+\.\d+
+      annotations:
+        fork-per-release: "true"
+      always_run: true
+      optional: false
```

Comment:
and not be optional , even in the begining ?

Reply:
May be better for debugging, changing.